Device: cuda
/home/lauren/nephronetwork/models/siamese_network/
/storage/ind_train_us_seq/
/storage/ind_val_us_seq/
TrainEpoch	0	ACC	49%	Loss	0.705057	AUC	0.496763	AUPRC	0.496083	TN	3872	FP	3638	FN	3949	TP	3541
ValEpoch	0	ACC	52%	Loss	0.692936	AUC	0.526891	AUPRC	0.514025	TN	164	FP	590	FN	128	TP	618
TrainEpoch	1	ACC	50%	Loss	0.693350	AUC	0.501975	AUPRC	0.500899	TN	4083	FP	3427	FN	4020	TP	3470
ValEpoch	1	ACC	50%	Loss	0.693182	AUC	0.504822	AUPRC	0.512514	TN	36	FP	718	FN	28	TP	718
TrainEpoch	2	ACC	49%	Loss	0.693412	AUC	0.499868	AUPRC	0.496782	TN	3758	FP	3752	FN	3760	TP	3730
ValEpoch	2	ACC	51%	Loss	0.693276	AUC	0.485106	AUPRC	0.499070	TN	688	FP	66	FN	665	TP	81
TrainEpoch	3	ACC	49%	Loss	0.693728	AUC	0.487307	AUPRC	0.485112	TN	3120	FP	4390	FN	3219	TP	4271
ValEpoch	3	ACC	50%	Loss	0.693131	AUC	0.508616	AUPRC	0.510627	TN	751	FP	3	FN	742	TP	4
TrainEpoch	4	ACC	50%	Loss	0.693443	AUC	0.502138	AUPRC	0.497923	TN	3955	FP	3555	FN	3888	TP	3602
ValEpoch	4	ACC	49%	Loss	0.693759	AUC	0.479703	AUPRC	0.493019	TN	0	FP	754	FN	1	TP	745
TrainEpoch	5	ACC	49%	Loss	0.693276	AUC	0.497953	AUPRC	0.495892	TN	2908	FP	4602	FN	2903	TP	4587
ValEpoch	5	ACC	50%	Loss	0.693781	AUC	0.490062	AUPRC	0.487143	TN	754	FP	0	FN	746	TP	0
TrainEpoch	6	ACC	49%	Loss	0.693418	AUC	0.502104	AUPRC	0.497591	TN	5277	FP	2233	FN	5281	TP	2209
ValEpoch	6	ACC	49%	Loss	0.693217	AUC	0.527904	AUPRC	0.511096	TN	1	FP	753	FN	1	TP	745
TrainEpoch	7	ACC	49%	Loss	0.693439	AUC	0.498457	AUPRC	0.497760	TN	3357	FP	4153	FN	3368	TP	4122
ValEpoch	7	ACC	50%	Loss	0.693323	AUC	0.495762	AUPRC	0.487742	TN	98	FP	656	FN	80	TP	666
TrainEpoch	8	ACC	49%	Loss	0.693518	AUC	0.496031	AUPRC	0.493163	TN	3332	FP	4178	FN	3389	TP	4101
ValEpoch	8	ACC	49%	Loss	0.693231	AUC	0.495268	AUPRC	0.488325	TN	400	FP	354	FN	403	TP	343
TrainEpoch	9	ACC	50%	Loss	0.693284	AUC	0.503297	AUPRC	0.499642	TN	4544	FP	2966	FN	4508	TP	2982
ValEpoch	9	ACC	49%	Loss	0.693581	AUC	0.500579	AUPRC	0.496254	TN	0	FP	754	FN	0	TP	746
TrainEpoch	10	ACC	49%	Loss	0.693370	AUC	0.495015	AUPRC	0.499860	TN	4803	FP	2707	FN	4809	TP	2681
ValEpoch	10	ACC	49%	Loss	0.693621	AUC	0.507038	AUPRC	0.500424	TN	1	FP	753	FN	0	TP	746
TrainEpoch	11	ACC	49%	Loss	0.693487	AUC	0.496551	AUPRC	0.499050	TN	3794	FP	3716	FN	3850	TP	3640
ValEpoch	11	ACC	49%	Loss	0.693365	AUC	0.500971	AUPRC	0.494577	TN	34	FP	720	FN	43	TP	703
TrainEpoch	12	ACC	49%	Loss	0.693477	AUC	0.493160	AUPRC	0.497274	TN	2958	FP	4552	FN	2996	TP	4494
ValEpoch	12	ACC	49%	Loss	0.693745	AUC	0.474100	AUPRC	0.484714	TN	8	FP	746	FN	8	TP	738
TrainEpoch	13	ACC	50%	Loss	0.693294	AUC	0.497861	AUPRC	0.500258	TN	4468	FP	3042	FN	4451	TP	3039
ValEpoch	13	ACC	50%	Loss	0.692957	AUC	0.518670	AUPRC	0.519856	TN	724	FP	30	FN	707	TP	39
TrainEpoch	14	ACC	49%	Loss	0.693424	AUC	0.495238	AUPRC	0.493971	TN	3400	FP	4110	FN	3420	TP	4070
ValEpoch	14	ACC	50%	Loss	0.693090	AUC	0.506050	AUPRC	0.501103	TN	592	FP	162	FN	577	TP	169
TrainEpoch	15	ACC	50%	Loss	0.693444	AUC	0.499851	AUPRC	0.496202	TN	4211	FP	3299	FN	4164	TP	3326
ValEpoch	15	ACC	47%	Loss	0.693536	AUC	0.475430	AUPRC	0.483288	TN	117	FP	637	FN	146	TP	600
TrainEpoch	16	ACC	50%	Loss	0.693382	AUC	0.496165	AUPRC	0.494756	TN	4085	FP	3425	FN	4045	TP	3445
ValEpoch	16	ACC	50%	Loss	0.693180	AUC	0.501177	AUPRC	0.502641	TN	240	FP	514	FN	236	TP	510
TrainEpoch	17	ACC	49%	Loss	0.693552	AUC	0.486700	AUPRC	0.488301	TN	3449	FP	4061	FN	3544	TP	3946
ValEpoch	17	ACC	49%	Loss	0.693350	AUC	0.480526	AUPRC	0.489355	TN	662	FP	92	FN	665	TP	81
TrainEpoch	18	ACC	50%	Loss	0.693298	AUC	0.502798	AUPRC	0.500095	TN	4767	FP	2743	FN	4723	TP	2767
ValEpoch	18	ACC	48%	Loss	0.693340	AUC	0.485675	AUPRC	0.478693	TN	652	FP	102	FN	668	TP	78
TrainEpoch	19	ACC	49%	Loss	0.693460	AUC	0.495769	AUPRC	0.497992	TN	4062	FP	3448	FN	4129	TP	3361
ValEpoch	19	ACC	50%	Loss	0.693303	AUC	0.492102	AUPRC	0.484095	TN	743	FP	11	FN	739	TP	7
TrainEpoch	20	ACC	49%	Loss	0.693374	AUC	0.495284	AUPRC	0.494944	TN	3930	FP	3580	FN	3971	TP	3519
ValEpoch	20	ACC	50%	Loss	0.693284	AUC	0.518523	AUPRC	0.515243	TN	754	FP	0	FN	746	TP	0
TrainEpoch	21	ACC	49%	Loss	0.693320	AUC	0.495647	AUPRC	0.495135	TN	4040	FP	3470	FN	4072	TP	3418
ValEpoch	21	ACC	50%	Loss	0.693133	AUC	0.507265	AUPRC	0.509163	TN	279	FP	475	FN	267	TP	479
^CTraceback (most recent call last):
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 166, in try_read
    return read_wrapper(lambda f: pkl.load(f))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 148, in read_wrapper
    return func(f)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 166, in <lambda>
    return read_wrapper(lambda f: pkl.load(f))
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 5: ordinal not in range(128)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 173, in try_read
    lambda f: pc.load(f, encoding=encoding, compat=False))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 148, in read_wrapper
    return func(f)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 173, in <lambda>
    lambda f: pc.load(f, encoding=encoding, compat=False))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/compat/pickle_compat.py", line 227, in load
    return up.load()
  File "/usr/lib/python3.5/pickle.py", line 1039, in load
    dispatch[key[0]](self)
  File "/usr/lib/python3.5/pickle.py", line 1177, in load_binstring
    self.append(self._decode_string(data))
  File "/usr/lib/python3.5/pickle.py", line 1159, in _decode_string
    return value.decode(self.encoding, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 5: ordinal not in range(128)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 179, in read_pickle
    return try_read(path)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 177, in try_read
    lambda f: pc.load(f, encoding=encoding, compat=True))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 148, in read_wrapper
    return func(f)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 177, in <lambda>
    lambda f: pc.load(f, encoding=encoding, compat=True))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/compat/pickle_compat.py", line 227, in load
    return up.load()
  File "/usr/lib/python3.5/pickle.py", line 1039, in load
    dispatch[key[0]](self)
  File "/usr/lib/python3.5/pickle.py", line 1177, in load_binstring
    self.append(self._decode_string(data))
  File "/usr/lib/python3.5/pickle.py", line 1159, in _decode_string
    return value.decode(self.encoding, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 5: ordinal not in range(128)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 166, in try_read
    return read_wrapper(lambda f: pkl.load(f))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 148, in read_wrapper
    return func(f)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 166, in <lambda>
    return read_wrapper(lambda f: pkl.load(f))
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 5: ordinal not in range(128)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train_triamese_network.py", line 319, in <module>
    main()
  File "train_triamese_network.py", line 316, in main
    train(args,triad_training_dataloader, triad_valid_dataloader,max_epochs,SiamNet,process_results)
  File "train_triamese_network.py", line 161, in train
    for batch_idx, (data, target) in enumerate(training_generator):
  File "/home/lauren/.local/lib/python3.5/site-packages/torch/utils/data/dataloader.py", line 615, in __next__
    batch = self.collate_fn([self.dataset[i] for i in indices])
  File "/home/lauren/.local/lib/python3.5/site-packages/torch/utils/data/dataloader.py", line 615, in <listcomp>
    batch = self.collate_fn([self.dataset[i] for i in indices])
  File "train_triamese_network.py", line 97, in __getitem__
    imgs,target = load_sample(self.filelist[index])
  File "train_triamese_network.py", line 70, in load_sample
    in_df = pd.read_pickle(infile)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 182, in read_pickle
    return try_read(path, encoding='latin1')
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 173, in try_read
    lambda f: pc.load(f, encoding=encoding, compat=False))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 148, in read_wrapper
    return func(f)
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/io/pickle.py", line 173, in <lambda>
    lambda f: pc.load(f, encoding=encoding, compat=False))
  File "/home/lauren/.local/lib/python3.5/site-packages/pandas/compat/pickle_compat.py", line 227, in load
    return up.load()
  File "/usr/lib/python3.5/pickle.py", line 1039, in load
    dispatch[key[0]](self)
  File "/usr/lib/python3.5/pickle.py", line 1177, in load_binstring
    self.append(self._decode_string(data))
  File "/usr/lib/python3.5/pickle.py", line 1159, in _decode_string
    return value.decode(self.encoding, self.errors)
KeyboardInterrupt
